Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TestSuiteEntity>
+   <description>Submit appeal with invalid denial code from Public link</description>
+   <name>Appeal Denied License Application - Invalid denial code</name>
+   <tag></tag>
+   <isRerun>false</isRerun>
+   <mailRecipient></mailRecipient>
+   <numberOfRerun>3</numberOfRerun>
+   <pageLoadTimeout>30</pageLoadTimeout>
+   <pageLoadTimeoutDefault>true</pageLoadTimeoutDefault>
+   <rerunFailedTestCasesOnly>false</rerunFailedTestCasesOnly>
+   <rerunImmediately>true</rerunImmediately>
+   <testSuiteGuid>32164761-29a0-4405-add3-11d985eab8b5</testSuiteGuid>
+   <testCaseLink>
+      <guid>1d9c2ddd-2cc3-4f63-bf25-884bde29c512</guid>
+      <isReuseDriver>false</isReuseDriver>
+      <isRun>true</isRun>
+      <testCaseId>Test Cases/Licensure Unit License Application/Appeal Denied License Application - Invalid denial code</testCaseId>
```

Comment:
Please, move the Test Suite to Individual Test Cases folder. The name of the Test Suite should be the same of the Test Case.

Reply:
Done